src/components/MetaMaskConnection/MetaMaskConnection.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useEffect } from "react";
+import ConnectButton from "./components/ConnectButton";
+import ConnectedMenu from "./components/ConnectedMenu";
+import { ethereum } from "../../common/types";
+
+async function checkCorrectNetwork(chainId: string) {
```

Comment:
@gavinspr are you using ```chainId``` here? maybe to check if you're on the right chain already?

Reply:
Ah yeah I didn't  need to pass that in, I removed it. 